.chloggen/fix_servicegraph-windows-tests.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
```

Comment:
This change doesn't require a change log: it is not visible to users. You can remove this file from the change I will add the label to skip the check for a change log.

Reply:
Done!